corehq/form_processor/tests/test_basic_cases.py

```diff
@@ -0,0 +1,203 @@
+from datetime import datetime
+import uuid
+from django.conf import settings
+from django.test import TestCase
+from casexml.apps.case.mock import CaseBlock
+from corehq.form_processor.interfaces.processor import FormProcessorInterface
+from corehq.form_processor.test_utils import FormProcessorTestUtils, run_with_all_backends
+
+DOMAIN = 'fundamentals'
+
+
+class FundamentalCaseTests(TestCase):
```

Comment:
I'm sure these are covered in other tests but I wanted some simple tests to just verify the basics of case processing